tox.ini

```
@@ -1,16 +1,6 @@
 [tox]
 envlist =
-    test{,-jwst}{,-devdeps}-xdist{,-cov}{,-nocrds}
-    build-{docs,dist}
-
-# tox environments are constructed with so-called 'factors' (or terms)
-# separated by hyphens, e.g. test-devdeps-cov. Lines below starting with factor:
-# will only take effect if that factor is included in the environment name. To
-# see a list of example environments that can be run, along with a description,
-# run:
-#
-#     tox -l -v
-#
+    py3{,11,12}{,-jwst}{,-devdeps}{,-cov}{,-nocrds}-xdist
```

```suggestion
    py3{,11,12}{,-jwst}{,-devdeps}{,-cov}{,-nocrds}{,-xdist}
```
since `-xdist` is optional.